frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -135,64 +98,99 @@ const Group: React.FC<GroupProps> = ({
+    setListOpen(!openList);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setListOpen(false);
+  };
+
+  let headingPath = window.location.pathname.replace("/", "").split("/")[0];
+
+  if (headingPath === "ec2") {
+    headingPath = "aws";
+  }
+
+  const isSelected = headingPath == heading.toLowerCase();
+
   // n.b. if a Workflow Grouping has no workflows in it don't display it even if
   // it's not explicitly marked as hidden.
   if (childrenList.length === 0) {
     return null;
   }
+
   return (
-    <List data-qa="workflowGroup">
-      <ListItem button onClick={() => updateOpenGroup(heading)}>
-        <GroupHeading color="primary" variant="h6">
-          {heading}
-        </GroupHeading>
-        <Grid container justify="flex-end" data-qa="toggle">
-          {open ? <ExpandLessIcon /> : <ExpandMoreIcon />}
-        </Grid>
-      </ListItem>
-      {!open ? <Divider variant="middle" /> : null}
-      <Collapse in={open} timeout="auto" unmountOnExit>
-        <List component="div" disablePadding>
-          {childrenList.map((c: React.ReactElement) => {
-            return React.cloneElement(c, { onClick: onNavigate });
-          })}
-        </List>
-      </Collapse>
-    </List>
+    <GroupList
+      data-qa="workflowGroup"
+      ref={anchorRef}
+      aria-controls={openList ? "workflow-options" : undefined}
+      aria-haspopup="true"
+      onClick={handleToggle}
+    >
+      <GroupListItem button onClick={() => updateOpenGroup(heading)} selected={isSelected}>
+        <GroupHeading align="center">{heading}</GroupHeading>
+        <Collapse in={open} timeout="auto" unmountOnExit>
+          <Popper
+            open={openList}
+            anchorEl={anchorRef.current}
+            role={undefined}
+            transition
+            placement="right-start"
+            style={{ zIndex: 1200 }}
+          >
+            <Paper>
+              <ClickAwayListener onClickAway={handleClose}>
+                <List component="div" disablePadding id="workflow-options">
+                  {childrenList.map((c: React.ReactElement) => {
+                    return React.cloneElement(c, { onClick: onNavigate });
+                  })}
+                </List>
+              </ClickAwayListener>
+            </Paper>
+          </Popper>
+        </Collapse>
+      </GroupListItem>
+    </GroupList>
   );
 };
 
 interface LinkProps {
   to: string;
   text: string;
   onClick: () => void;
-  trending?: boolean;
 }
 
-const Link: React.FC<LinkProps> = ({ to, text, onClick, trending = false }) => {
-  const theme = useTheme();
+const Link: React.FC<LinkProps> = ({ to, text, onClick }) => {
   const isSelected = window.location.pathname.replace("/", "") === to;
-  const selectedStyle = isSelected ? { color: theme.palette.accent.main } : {};
   return (
-    <ListItem
-      component={NavigationLink}
+    <LinkListItem
```

Comment:
I'll clean up the use of isSelectedWorkflow. essentially want to change the background and text color if a workflow is chosen based on the url of the browser